Dockerfile

```diff
@@ -43,30 +47,35 @@ RUN tar -cvf valhalla.debug.tar valhalla_*.debug && gzip -9 valhalla.debug.tar
 RUN rm -f valhalla_*.debug
 RUN strip --strip-debug --strip-unneeded valhalla_* || true
 RUN strip /usr/local/lib/libvalhalla.a
-RUN strip /usr/lib/python3/dist-packages/valhalla/python_valhalla*.so
+RUN strip /usr/local/venv/lib/python3.11/site-packages/valhalla/python_valhalla*.so
 
 ####################################################################
 # copy the important stuff from the build stage to the runner image
-FROM ubuntu:22.04 as runner
+FROM ubuntu:23.04 as runner
 MAINTAINER Kevin Kreiser <kevinkreiser@gmail.com>
 
+# basic paths
+ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$PATH
+ENV LD_LIBRARY_PATH=/usr/local/lib:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/lib32:/usr/lib32
+
 # github packaging niceties
 LABEL org.opencontainers.image.description = "Open Source Routing Engine for OpenStreetMap and Other Datasources"
 LABEL org.opencontainers.image.source = "https://github.com/valhalla/valhalla"
 
+# grab the builder stages artifacts
 COPY --from=builder /usr/local /usr/local
-COPY --from=builder /usr/lib/python3/dist-packages/valhalla/* /usr/lib/python3/dist-packages/valhalla/
```

Comment:
not entirely sure about this. is the installed package the exact same as the built package in /usr/local? maybe..

Reply:
Oh I see.. it's installed to /usr/local in the venv

Reply:
yeah